src/js/common/schemaform/definitions/address.js

```diff
@@ -30,14 +48,69 @@ export function schema(currentSchema, isRequired = false) {
   };
 }
 
-export function uiSchema(label = 'Address') {
+export function uiSchema(label = 'Address', useStreet3 = false) {
+  let fieldOrder = ['country', 'street', 'street2', 'street3', 'city', 'state', 'postalCode'];
+  if (!useStreet3) {
+    fieldOrder = fieldOrder.filter(field => field !== 'street3');
+  }
+
   return {
     'ui:title': label,
-    'ui:field': 'address',
+    // 'ui:field': 'address',
```

Comment:
I just need to remove it entirely. I commented out when I first started this work.